x-pack/plugins/ingest_pipelines/public/application/components/pipeline_form/pipeline_test_flyout/tabs/tab_output.tsx

```diff
@@ -59,7 +59,7 @@ export const OutputTab: React.FunctionComponent<Props> = ({
         <p>
           <FormattedMessage
             id="xpack.ingestPipelines.testPipelineFlyout.outputTab.descriptionText"
-            defaultMessage="The output of the executed pipeline. {runLink}"
+            defaultMessage="View the final output data, or see how each processor affects the ingest document as it passes through the pipeline. {runLink}"
```

Comment:
@gchaps Let me know what you think of this (screenshot in initial PR comment). This differs from your original suggestion, as the `View verbose output` is actually disabled by default.